packages/shield/packaging

```diff
@@ -5,11 +5,13 @@ CPUS=$(grep -c ^processor /proc/cpuinfo)
 mkdir -p ${BOSH_INSTALL_TARGET}/bin
 tar -xzf shield/shield-server-linux-amd64*.tar.gz
 pushd shield-server-linux-amd64*/
-  cp agent/shield-agent   ${BOSH_INSTALL_TARGET}/bin
-  cp cli/shield           ${BOSH_INSTALL_TARGET}/bin
-  cp daemon/shieldd       ${BOSH_INSTALL_TARGET}/bin
-  cp daemon/shield-schema ${BOSH_INSTALL_TARGET}/bin
-  cp daemon/shield-pipe   ${BOSH_INSTALL_TARGET}/bin
-  cp -R webui             ${BOSH_INSTALL_TARGET}/webui
-  cp -R plugins           ${BOSH_INSTALL_TARGET}/plugins
+  cp agent/shield-agent    ${BOSH_INSTALL_TARGET}/bin
+  cp cli/shield            ${BOSH_INSTALL_TARGET}/bin
+  cp daemon/shieldd        ${BOSH_INSTALL_TARGET}/bin
+  cp daemon/shield-schema  ${BOSH_INSTALL_TARGET}/bin
+  cp daemon/shield-migrate ${BOSH_INSTALL_TARGET}/bin
+  cp crypter/shield-crypt ${BOSH_INSTALL_TARGET}/bin
```

Comment:
can you line this one up?

also, shouldn't this be in `daemon`?

Reply:
Well its part of shield-pipe which runs on the agents so it ended up getting its own folder in artifacts. (Cleaned up the whitespace issue as well)